docs/en/admins/02_Prerequisites.md

```diff
@@ -7,8 +7,9 @@ You need to verify that your server can run FreshRSS before installing it. If yo
 | Software      | Recommended             | Also Works With         |
 | ------------- | ----------------------- | ----------------------- |
 | Web server    | **Apache 2.4+**         | nginx, lighttpd         |
-| PHP           | **PHP 7.4+**            |                         |
+| PHP           | **PHP 7.4+**            | Soft require Apache 2.4+ (but repair minimal compatibility with Apache 2.2) |
```

Comment:
Most of that is part of the commit message which shouldn't be copied here verbatim.

I'm not sure if the 2.2 thing should be mentioned, but if so perhaps something like:
```suggestion
| PHP           | **PHP 7.4+**            | Apache 2.4+ (minimal compatibility with Apache 2.2) |
```

Reply:
Or perhaps simply: `2.2+, 2.4+ recommended`

Reply:
It is just copy&pasted from the changelog of 1.23.0 https://github.com/FreshRSS/FreshRSS/releases/tag/1.23.0

Reply:
Exactly ;-)

Reply:
by the way: I see that the notice is placed in the wrong row.... -.-